### PR TITLE
Improve SEO discoverability for author

### LINKED ISSUE #4752: Pass correct props to article schema to enable SEO discoverability
**Is your feature request related to a problem? Please describe.**
As an SEO expert I want to ensure the correct values are being passed to the Schema, so that my page is discoverable and ranked in search. 

**Describe the solution you'd like**
At the moment when you use the https://search.google.com/structured-data/testing-tool and test a test MAP page like: https://www.test.bbc.com/pidgin/23248703
  you will see that we're not setting values the following fields: 
Screenshot 2019-11-20 at 11 26 32

We need to set the correct props for the Article and Author objects and fields: 

| Object Name | Field Name | Value Example | Is Dynamic value? | Source |
| -- | -- | -- | -- |-- |
|Article|datePublished | 2019-11-21T19:35:04+01:00 | Yes | Ares - firstPublished | 
||dateModified | 2019-11-21T19:35:04+01:00 | Yes | Ares - timestamp |
|| headline | BBC Pidgin Minute | Yes | Ares - headline | 
| Author | @type | NewsMediaOrganisation or Organization (if trust project participant) | No | Simorgh | 
| | name | BBC News Pidgin | Yes (per service) | Simorgh (same as the schema publisher name) |
| | noBylinesPolicy | http://www.bbc.co.uk/news/help-41670342#authorexpertise | No | Simorgh (check with articles) |

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by karinathomasbbc**
Resolves #4752

**Overall change:**
Ensure article props are set correctly, to improve SEO discoverability

**Code changes:**
- default showAuthor to true so that author always appears

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by ryanmccombe — APPROVED** [review]
Do we want "author" added by default to Live Radio and front pages too?  I thought we only wanted it for MAPs and Articles - I've been out of the loop though!

Looks good aside from that question

**Review by pharingee — APPROVED** [review]

**Comment by karinathomasbbc** [comment]
> Do we want "author" added by default to Live Radio and front pages too? I thought we only wanted it for MAPs and Articles - I've been out of the loop though!
> 
> Looks good aside from that question

Good catch! It is only Articles & MAPs which should have author added.  I have updated the diff accordingly.

File: src/app/containers/CpsAssetPageMain/index.test.jsx [hunk 1]
```diff
@@ -101,11 +101,14 @@ describe('CpsAssetPageMain', () => {
...
 
+    it('should render component', () => {
+      expect(asFragment()).toMatchSnapshot();
+    });
+
     it('should render paragraph', () => {
       const paragraphText = getBlockTextAtIndex(1, pidginPageData);
 
       expect(getByText(escapedText(paragraphText))).toBeInTheDocument();
-      expect(asFragment()).toMatchSnapshot();
```
Comment (ryanmccombe) [inline src/app/containers/CpsAssetPageMain/index.test.jsx:108]: giphy

**Review by chris-hinds — APPROVED** [review]
👍

**Pull request merged**